selfdrive/monitoring/driver_monitor.py

```diff
@@ -18,10 +18,10 @@ class DRIVER_MONITOR_SETTINGS():
   def __init__(self, DT_DMON=DT_DMON):
     self._DT_DMON = DT_DMON
     # ref (page15-16): https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:42018X1947&rid=2
-    self._AWARENESS_TIME = 30. # passive wheeltouch total timeout
-    self._AWARENESS_PRE_TIME_TILL_TERMINAL = 15.
-    self._AWARENESS_PROMPT_TIME_TILL_TERMINAL = 6.
-    self._DISTRACTED_TIME = 11. # active monitoring total timeout
+    self._AWARENESS_TIME = 99999. # passive wheeltouch total timeout
```

Comment:
against commas safety. cannot merge this

Reply:
only for test i will change this